### PR TITLE
GossipingPropertyFileSnitch & Replace node fix

### PR DESCRIPTION
**Pull request by sbonde**
1. cassandra-topology.properties fix for GossipingPropertyFileSnitch
2. In  replace', JVM option added in cassandra-env.sh instead of starting with option and then removed from cassandra-env.sh after node replaced & started.

### CODE REVIEW

File: components/cookbooks/cassandra/templates/default/cassandra-topology.properties.erb [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+<% if node.workorder.rfcCi.ciAttributes.endpoint_snitch =~ /\.GossipingPropertyFileSnitch/ 
+dc_rack_map = JSON.parse(node.workorder.rfcCi.ciAttributes.cloud_dc_rack_map) 
+cloud_name = node[:workorder][:cloud][:ciName]
+dc_rack = dc_rack_map[cloud_name] 
+	if dc_rack.nil?
+	      Chef::Log.error("missing dc_rack_map entry for: #{cloud_name}")
+	      exit 1
+	end %>
+default=<%= dc_rack %> 
+fe80\:0\:0\:0\:202\:b3ff\:fe1e\:8329=<%= dc_rack %>
```
Comment (schwankl) [inline components/cookbooks/cassandra/templates/default/cassandra-topology.properties.erb]: whats this?

Comment (sbonde) [inline components/cookbooks/cassandra/templates/default/cassandra-topology.properties.erb]: Was keeping default for GossipingPropertyFileSnicth. But as cassandra-topology.properties is not required for GossipingPropertyFileSnicth, removing if condition.
Also modified topology.rb to remove cassandra-topology.properties for GossipingPropertyFileSnicth

**Comment by gauravlall** [comment]
This can be merged could you please  resolve conflicts

**Comment by sbonde** [comment]
@gauravlall - I have fixed the outdated version. Could you please merge it?

**Pull request merged**